### PR TITLE
Tags Tab UI

### PR DESCRIPTION
**Pull request by itaiad200**

### CODE REVIEW

File: webui/src/pages/repositories/repository/tags.jsx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                            <span className="ml-2 mr-2">@</span>
+                            <RefDropdown
+                                repo={repo}
+                                emptyText={'Select Branch'}
+                                selected={selectedBranch}
+                                selectRef={(refId) => {
+                                    setSelectedBranch(refId);
+                                }}
+                                withCommits={true}
+                                withWorkspace={false} />
```
Comment (itaiad200) [inline webui/src/pages/repositories/repository/tags.jsx:135]: I can't filter branches using this dropdown, is it on purpose?

File: webui/src/pages/repositories/repository/tags.jsx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        <ListGroup.Item>
+            <div className="clearfix">
+                <div className="float-left">
+                    <h6>
+                        <Link href={{
+                            pathname: '/repositories/:repoId/objects',
+                            params: { repoId: repo.id },
+                            query: { ref: tag.commit_id }
+                        }}>
+                            {tag.id}
```
Comment (itaiad200) [inline webui/src/pages/repositories/repository/tags.jsx:47]: Could we add tip when the user hover that is something like:
"Explore objects at the tag"

**Review by itaiad200 — COMMENTED** [review]
Thanks @shamikakumar ! some great work with the tags page!
See the comments below. Not all tasks in the issue are implemented in this PR, which is fine for this stage.

**Comment by itaiad200** [comment]
Hey @shamikakumar, thanks for your efforts with this one! We plan on releasing this feature as part of the next release, when do you think you'll have the time to complete it?

**Comment by shamikakumar** [comment]
hey @itaiad200 I'm facing issues trying to run the project locally, could you help me out

**Comment by itaiad200** [comment]
> hey @itaiad200 I'm facing issues trying to run the project locally, could you help me out

Sure @shamikakumar , what seems to be the problem?
Also - sent you a Slack message if you feel IM is more suitable.

File: webui/src/pages/repositories/repository/tags.jsx (hunk 1 above)
Comment (shamikakumar) [reply webui/src/pages/repositories/repository/tags.jsx:135]: Hey @itaiad200 not able to figure out what's causing this. I'm not able to use the filter option in the Create Branch modal in the branches page either

Comment (itaiad200) [reply webui/src/pages/repositories/repository/tags.jsx:135]: True, I can't either. Unrelated to this PR.
Opened #2460 and resolving this comment.

**Pull request closed without merging**